vaadin-cdi/src/main/java/com/vaadin/cdi/CdiVaadinServletService.java

```diff
@@ -146,21 +161,17 @@ public CdiVaadinServlet getServlet() {
      */
     public static class CdiVaadinServiceDelegate implements Serializable {
 
-        private final VaadinService vaadinService;
-
         private transient BeanManager beanManager;
 
         private final UIEventListener uiEventListener;
 
         public CdiVaadinServiceDelegate(VaadinService vaadinService,
```

Comment:
Could we remove from here as well the `vaadinService`, if I am correct it is unused now.

Reply:
In that case probably from this line as well: 
`this.delegate = new CdiVaadinServiceDelegate(this, beanManager);`

Reply:
done

Reply:
Thank you vm! 🙇